CHANGELOG.md

```diff
@@ -91,6 +91,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 
 - `cardano-serialization-lib` has been updated to `v13.2.0` ([#1656](https://github.com/Plutonomicon/cardano-transaction-lib/pull/1656))
+- `QueryHandle` has been renamed to `Provider`
```

Comment:
Each changelog entry should include a link to the corresponding pull request or issue. See other entries for reference.